pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>software.amazon.payloadoffloading</groupId>
             <artifactId>payloadoffloading-common</artifactId>
-            <version>1.0.0</version>
+            <version>2.0.0</version>
         </dependency>
         <dependency>
             <groupId>com.amazonaws</groupId>
```

Comment:
note that until `amazon-sqs-java-extended-client-lib` is updated to use aws sdk v2, there will still be some references to aws sdk v1 in the code. i put up a PR to update the sqs sdk as well (https://github.com/awslabs/amazon-sqs-java-extended-client-lib/pull/58)

Reply:
updated to `amazon-sqs-java-extended-client-lib` v2.0.2